src/box/lua/config/instance_config.lua

```diff
@@ -2475,6 +2475,12 @@ return schema.new('instance_config', schema.record({
             type = 'string',
         }),
     }),
+    isolated = schema.scalar({
+        type = 'boolean',
+        default = false,
+        -- TODO: verify that the option is set in instance scope
+        -- or using an environment variable.
+    }),
```

Comment:
I'm going to add the validation. It is easy now, when we have computed annotations.

Reply:
Done.